app/grandchallenge/cases/serializers.py

```diff
@@ -166,7 +173,7 @@ def validate(self, attrs):
             raise ValidationError("Filenames must be unique")
 
         num_targets = sum(f in attrs for f in self.targets)
-        if num_targets > 1:
+        if num_targets > 2 and "interface" not in self.targets:
```

Comment:
The logic here needs to be reworked - interface can only be set if archive it set, right?